pkg/network/event_usm_common.go

```diff
@@ -13,5 +13,6 @@ const (
 	ProtocolHTTP
 	ProtocolHTTP2
 	ProtocolTLS
-	MaxProtocols
+	ProtocolAMQP = 8
```

Comment:
we adding a gap between TLS and AMQP id ?

Reply:
the list here represents the same list in `agent-payload` and there we have other protocol in that gap.
each PR adds the protocol it represents

Reply:
why not using type aliasing here with agent payload ProtocolType
type ProtocolType agentpb.ProtocolType
const (
    ProtocolHTTP2 = ProtocolType_protocolHTTP2
...
)

Reply:
won't work on windows